trans_sec/controller/ddos_sdn_controller.py

```diff
@@ -45,14 +56,57 @@ def start(self, add_di):
         logger.info('Starting HTTP server on port - [%s]',
                     self.http_server.port)
         self.http_server.start()
-
+        self.create_drop_report()
         self.__main_loop()
 
     def stop(self):
         logger.info('Stopping SDN controller')
         self.running = False
         self.http_server.stop()
 
+    def create_drop_report(self):
+        for controller in self.controllers.values():
```

Comment:
You could add a call to the core_controller here (that calls a new method on bfruntime_lib.core_switch.CoreSwitch#get_telem_rtp_ip(udp_port) that obtains the IPv4 or 6 vaule for a given UDP value. If none, skip...